src/main/resources/assets/admin/common/js/form/inputtype/support/BaseInputTypeNotManagingAdd.ts

```diff
@@ -357,6 +341,16 @@ export abstract class BaseInputTypeNotManagingAdd
             const inputEl: Element = occurrenceView.getInputElement();
             this.validateUserInput(inputEl);
             this.validateRequiredContract(inputEl, value);
+            this.validateCustomErrors(occurrenceView);
         }
     }
+
+    protected validateCustomErrors(occurrenceView: InputOccurrenceView): void {
+        this.getContext().formContext.getValidationErrors().forEach((error: ValidationError) => {
+            if (occurrenceView.getDataPath().asRelative().toString() === error.getPropertyPath()) {
```

Comment:
`occurrenceView.getDataPath().asRelative().toString()` should be calculated once, outside the forEach loop.